src/cr/cube/dimension.py

```diff
@@ -381,6 +381,16 @@ def prune(self):
             return True
         return False
 
+    @lazyproperty
+    def selected_categories(self):
+        """List of selected categories specified for this dimension."""
+        selected_categories = self._dimension_dict["references"].get(
+            "selected_categories"
+        )
+        if not selected_categories or selected_categories == [{}]:
+            return []
+        return selected_categories
```

Comment:
In general, a lazyproperty should not return a mutable object. Prefer tuple for sequence return types.